README.md

```diff
@@ -11,8 +11,26 @@ organisation of neuroimaging data.
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-**Want to learn more about working with BIDS? Have a question, comment, or suggestion?**
-Open or comment on one of our [NeuroStars issues](https://neurostars.org/tags/bids) or check out the [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit)!
+# Using BIDS
+
+To use BIDS, all you need is neuro data, a computer, and the
+[BIDS specification](https://bids-specification.readthedocs.io/en/stable/).
+
+As a user, the GitHub repository and all files therein can be safely ignored
+to focus on the rendered content instead (see link above).
+The specification is provided in the form of a webpage, built using
+[MkDocs](https://www.mkdocs.org/) and [Read the Docs](https://readthedocs.org/).
+
+# Asking Questions
+
+If you have a question about BIDS, you have several options:
+
+1. Read some introductory material, most likely the very basic problems have already been addressed!
```

Comment:
Maybe we should actually move the Starter Kit under the previous section, since it has a lot of helpful information.

Reply:
Personally, I like it better to have:
1. what you need to use BIDS (only the spec, data, a computer)
2. how you can get help to get started (list of things, first of which is the starter kit)

it keeps the sections non-overlapping this way

Reply:
I think we might want to change the title of this section, then, only since they way the Starter Kit is described isn't as a place to _ask_ questions so much as a resource for many common user questions.

The others seem to really be avenues for folks to directly ask questions about BIDS Standard. Do you see what I mean ?

Reply:
Yes, I see your point :thinking: how about a simple "Getting started" instead of "Asking questions"?

Or do you have a different suggestion?